models/double_entry_transactions/int_quickbooks__deposit_double_entry.sql

```diff
@@ -28,6 +28,7 @@ uf_accounts as (
 
     where account_sub_type = 'UndepositedFunds'
```

Comment:
@fivetran-joemarkiewicz must have missed this the first time I released this, but what's the reason we use `account_sub_type` here and `account_type` everywhere else?

Reply:
Great question! This is because the deposit records will always specifically be tied to the UndepositedFunds account sub type. There are a multitude of Other Current Asset account types; however, for deposits we only want to associate the entries with the UndepostedFunds sub type of Other Current Assets.

Likewise, the other double entry checks are to account types that typically only have one version of the account type. For example, the Accounts Payable account type doesn't typically have other multiple account sub types. Therefore, we can rely on just the account type for those records.

Reply:
In theory, we could update the package to all just pull from the `account_sub_type` field, but this has not caused any issues and I would be more cautious to make a change like that now as we are unaware of the possible consequences.